### ISSUE TITLE
MOIU._hasvar removed from MOI

### ISSUE THREAD

**Issue by matbesancon** [JuliaOpt/Dualization.jl#74]
Error at the model creation step, `_hasvar` seems used, but is not (no longer?) defined in MOIU

```
ERROR: UndefVarError: _hasvar not defined
Stacktrace:
 [1] get_indices_variables(::MathOptInterface.ScalarAffineFunction{Float64}, ::Array{MathOptInterface.VariableIndex,1}) at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/objective_coefficients.jl:103
 [2] remove_variables at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/objective_coefficients.jl:92 [inlined]
 [3] remove_variables(::Dualization.PrimalObjective{Float64}, ::Array{MathOptInterface.VariableIndex,1}) at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/objective_coefficients.jl:88
 [4] get_primal_objective(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::Array{MathOptInterface.VariableIndex,1}) at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/objective_coefficients.jl:82
 [5] dualize(::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::Dualization.DualProblem{Float64,MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}}, ::Dualization.DualNames, ::Array{MathOptInterface.VariableIndex,1}, ::Bool) at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/dualize.jl:33
 [6] #dualize#2 at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/dualize.jl:16 [inlined]
 [7] (::Dualization.var"#kw##dualize")(::NamedTuple{(:dual_names,),Tuple{Dualization.DualNames}}, ::typeof(Dualization.dualize), ::MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}, ::Dualization.DualProblem{Float64,MathOptInterface.Utilities.CachingOptimizer{MathOptInterface.AbstractOptimizer,MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Float64}}}}) at ./none:0
 [8] #dualize#3(::Dualization.DualNames, ::typeof(Dualization.dualize), ::Model) at /home/mbesancon/.julia/packages/Dualization/VYRFk/src/dualize.jl:74
```

**Comment by blegat** [comment]
Fixed on master

**Comment by blegat** [comment]
Released as v1.2
